lib/hooks/coveralls/templates/default.html.haml

```diff
@@ -0,0 +1,9 @@
+%p
+  %a{ href: payload.url }
+    #{payload.repo_name}
+  #{message_from_coverage_change(payload.coverage_change)}
+  %span.label{class: 'label-info'}#{two_places_of_float(payload.covered_percent)}%
+  on #{payload.branch} after commit
+  %ul
```

Comment:
Since the commit is always single, I would like to use either `<p>` or `<br>` instead of `<ul>`.

Like this (line 2): 
![selection_347](https://cloud.githubusercontent.com/assets/43346/11233815/110052fe-8e05-11e5-8321-b5e57f4254da.png)